codalab/worker_manager/aws_batch_worker_manager.py

```diff
@@ -46,19 +46,31 @@ def add_arguments_to_subparser(subparser):
             default='codalab-batch-cpu',
             help='Name of the AWS Batch job queue to use',
         )
+        subparser.add_argument(
+            '--job-filter',
+            type=str,
+            help=(
+                'Ignore jobs on the job queue if their job name does not '
```

Comment:
Could we write the comment in a positive way to avoid negation (e.g., Keep only the jobs that match...)?